src/test/java/net/sf/jabref/importer/fileformat/InspecImportTest.java

```diff
@@ -0,0 +1,109 @@
+            }
+        }
+    }
+
+    @Test
+    public void testIsRecognizedFormatReject() throws IOException {
+        List<String> testList = Arrays.asList("CopacImporterTest1.txt", "CopacImporterTest2.txt",
+                "IEEEImport1.txt", "IsiImporterTest1.isi", "IsiImporterTestInspec.isi", "IsiImporterTestWOS.isi",
+                "IsiImporterTestMedline.isi", "RisImporterTest1.ris", "InspecImportTestFalse.txt");
+
+        for (String str : testList) {
+            try (InputStream inStream = InspecImportTest.class.getResourceAsStream(str)) {
+                assertFalse(inspecImp.isRecognizedFormat(inStream));
+            }
+        }
+    }
+
+    @Test
+    public void testCompleteBibtexEntryOnJournalPaperImport() throws IOException {
+        try (InputStream inStream = InspecImportTest.class.getResourceAsStream("InspecImportTest2.txt")) {
+            List<BibEntry> entries = inspecImp.importEntries(inStream, new OutputPrinterToNull());
+            assertEquals(1, entries.size());
+            BibEntry entry = entries.get(0);
+
+            assertEquals("The SIS project : software reuse with a natural language approach", entry.getField("title"));
+            assertEquals("Prechelt, Lutz", entry.getField("author"));
+            assertEquals("1992", entry.getField("year"));
+            assertEquals("Abstrakt", entry.getField("abstract"));
+            assertEquals("key", entry.getField("keywords"));
+            assertEquals("10000", entry.getField("journal"));
+            assertEquals("20", entry.getField("pages"));
+            assertEquals("19", entry.getField("volume"));
+            assertEquals(BibtexEntryTypes.ARTICLE.getName().toLowerCase(), entry.getType());
+        }
+    }
+
+    @Test
+    public void importConferencePaperGivesInproceedings() throws IOException {
+        String testInput = "Record.*INSPEC.*\n" +
+                "\n" +
+                "RT ~ Conference-Paper";
+        try (InputStream inStream = new ByteArrayInputStream(testInput.getBytes())) {
+            List<BibEntry> entries = inspecImp.importEntries(inStream, new OutputPrinterToNull());
+            inStream.close();
+            assertEquals(1, entries.size());
+            BibEntry entry = entries.get(0);
+            assertEquals(BibtexEntryTypes.INPROCEEDINGS.getName().toLowerCase(), entry.getType());
+        }
+    }
+
+    @Test
+    public void importMiscGivesMisc() throws IOException {
+        String testInput = "Record.*INSPEC.*\n" +
+                "\n" +
+                "RT ~ Misc";
+        try (InputStream inStream = new ByteArrayInputStream(testInput.getBytes())) {
+            List<BibEntry> entries = inspecImp.importEntries(inStream, new OutputPrinterToNull());
+            assertEquals(1, entries.size());
+            BibEntry entry = entries.get(0);
+            assertEquals(BibtexEntryTypes.MISC.getName().toLowerCase(), entry.getType());
+        }
+    }
+
+    @Test
+    public void testGetFormatName() {
```

Comment:
"testGetFormatName failed" can be removed (it doesn't add any information...) same below